workflows/prognostic_c48_run/tests/test_regression.py

```diff
@@ -598,7 +611,9 @@ def completed_rundir(configuration, tmpdir_factory):
     tendency_dataset_path = tmpdir_factory.mktemp("tendencies")
 
     if configuration == ConfigEnum.predictor:
-        model = get_mock_predictor()
+        model = get_mock_predictor(
+            dQ1_tendency=1 / 86400, mask_lowest_level_outputs=True
```

Comment:
Is this change in behavior intentional? (I think the dQ1 tendency was zero before). I'm wondering whether the checksum changes are from this or from the change in how wind tendencies are applied, or both. I think it would be better to leave the behavior of this test unchanged if possible.

Reply:
As discussed in Slack, indeed it was both (I lumped in some changes that also allowed us to verify that the tendencies were being properly in the vertical).  We decided to revert the changes required to test the vertical orientation of tendency updates, punting that responsibility to the Python wrapper.  The checksum changes now are only due to the fact that wind tendency updates are now applied during the first iteration of the time loop (when they were not in the past).